<launch>
  <node name="cobot_dynamixel_driver" pkg="cobot_dynamixel_driver" type="cobot_dynamixel_driver_mx64" respawn="false" output="screen" >
    <remap from="cobot_dynamixel_driver/joint_states" to="joint_states"/>
    <remap from="cobot_dynamixel_driver/goal" to="goal"/>
    <param name="fake_joints" value="6"/>
    <param name="load_joint_name" value="true"/>
    <param name="setting_file" value="$(find cobot_dynamixel_driver)/src/settings_mx64.xml" />
  </node>
</launch>
